server/api/src/rsDataObjRepl.cpp

```diff
@@ -424,6 +425,11 @@ int dataObjCopy(
     int srcRemoteFlag{};
     int source_l1descInx = L1desc[_destination_l1descInx].srcL1descInx;
     int srcL3descInx = L1desc[source_l1descInx].l3descInx;
+
+    // copy data size from src catalog to destination
+    L1desc[_destination_l1descInx].dataSize = L1desc[source_l1descInx].dataObjInfo->dataSize;
+    //L1desc[source_l1descInx].dataSize = L1desc[source_l1descInx].dataObjInfo->dataSize;
```

Comment:
leftover?

Reply:
Left it in there in case someone decides it is needed in the fruture.  Can delete

Reply:
delete it if it's not needed today.   leave a helpful comment if there's something to say.

Reply:
Probably just left over.